### PR TITLE
parameterised the app_secret_key

### LINKED ISSUE #50: Provide a way to specify user API secret
It would be nice to have an option in ansible's provision.yaml file to specify user API secret used on admin API endpoints.

### PR DESCRIPTION
**Pull request by harshad16**
fixes #50

### CODE REVIEW

**Comment by goern** [comment]
please rename APP_SECRET_KEY to THOTH_APP_SECRET_KEY So that the naming is consistent

**Comment by harshad16** [comment]
Made the renaming change APP_SECRET_KEY -> THOTH_APP_SECRET_KEY.

File: openshift/secret-template.yaml [hunk 1]
```diff
@@ -22,12 +22,17 @@ objects:
...
       app: thoth-core
   stringData:
-    app-secret-key: "887a2b06a52fc26cd444344df0276dde756133a2fdf8178c95488fbdc73c69c4"
+    app-secret-key: ${THOTH_APP_SECRET_KEY}
     thoth-secret: ${THOTH_SECRET}
     ceph-key-id: ${THOTH_CEPH_KEY_ID}
     ceph-secret-key: ${THOTH_CEPH_SECRET_KEY}
 
 parameters:
+  - description: Secret key for Application connection
```
Comment (fridex) [inline openshift/secret-template.yaml]: Worth to be more descriptive - this secret is used by Flask.

**Comment by harshad16** [comment]
Redefined the description of the THOTH_APP_SECRET_KEY in secret_template.yaml. :+1:

**Pull request merged**